src/Configuration.php

```diff
@@ -93,6 +94,7 @@
         'finder',
     ];
     private const PHP_SCOPER_CONFIG = 'scoper.inc.php';
+    private const DEFAULT_SIGNING_ALGORITHM = Phar::SHA1;
```

Comment:
are we sure SHA1 is always available? asking, as for other algorithms, we check if it's available for our installation of Phar ext...

Reply:
Good question, worth checking even more so since I'd like to check if it's worth pushing for a better signing algorithm